pkg/lib/cockpit-components-shutdown.jsx

```diff
@@ -32,6 +32,26 @@ import "cockpit-components-shutdown.scss";
 
 const _ = cockpit.gettext;
 
+// FIXME: https://github.com/patternfly/patternfly-react/issues/5564
+// Copied from: https://github.com/patternfly/patternfly-react/blob/master/packages/react-core/src/components/TimePicker/TimePickerUtils.tsx
+// and slightly adjusted
+const validateTime = (time) => {
```

Comment:
There are now two code copies of this around -- do you think that will be done in PF-react proper, and these can go away entirely in the forseeable future? Or should we move this into a cockpit-component-timepicker?

Reply:
I can work on the PF4 repo so that we will not need it:
It's because we cant really know in our code if the inserted value is valid: https://github.com/patternfly/patternfly-react/issues/5564

Also their validator is always applied regardless of the input so I had to make sure it's the same.
See: https://github.com/patternfly/patternfly-react/issues/5565

So, I will move this to lib, it does not hurt and work on it soon regardless.